src/models/version.ts

```diff
@@ -67,6 +67,7 @@ export const SVersionPrototypeSettings = s.object({
   branchImage: s.optional(s.string()),
   avatar: s.optional(s.string()),
   password: s.optional(s.string()),
+  hasPassword: s.optional(s.boolean()),
```

Comment:
Prototype Share View requires this. We don't want to send the full password over, but we need some information to indicate that a password was set.